sources/packages/api/src-sql/EducationProgramsOfferings/Alter-education-program-offerings.sql

```diff
@@ -0,0 +1,73 @@
+-- Add year_of_study to education_programs_offerings table.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN IF NOT EXISTS year_of_study INT NOT NULL DEFAULT 1;
+
+COMMENT ON COLUMN sims.education_programs_offerings.year_of_study IS 'Year of study of the study period.';
+
+-- Add show_yr_of_study to education_programs_offerings table.
+ALTER TABLE
+    sims.education_programs_offerings
+ADD
+    COLUMN IF NOT EXISTS show_yr_of_study BOOLEAN;
+
+COMMENT ON COLUMN sims.education_programs_offerings.show_yr_of_study IS 'This value determins if we show year of study to students.';
```

Comment:
Typo on _determins_.
Please check if the spell checker is enabled for .SQL files.

![image](https://user-images.githubusercontent.com/61259237/142907221-98011b82-2e93-4427-9148-5f809b807810.png)

Reply:
It wasn't enabled for sql files in my visual studio. I will enable it and btw. typo fixed